nuget.config

```diff
@@ -5,6 +5,7 @@
 
     <!-- do not add new sources to this file. If new packages are required, push them to one of these feeds. -->
     <add key="dotnetfeed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="dnceng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
```

Comment:
This is the new feed for Arcade/Helix. We still consume some packages from dotneed, so we need both.

Reply:
Is dotneed a typo or something I don't know about?

Reply:
Typo. I meant 'dotnetfeed'. :)